### PR TITLE
add loadpath cmd

### PR DESCRIPTION
**Pull request by duchaoyu**
- add proxy function in compas_ags.ags.loadpath
- add new cmd: AGS_check_loadpath

### CODE REVIEW

File: scripts/test_dof.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 from compas_ags.diagrams import FormGraph
 from compas_ags.diagrams import FormDiagram
-# from compas_ags.ags import graphstatics
+from compas_ags.ags import graphstatics
```
Comment (brgcode) [inline scripts/test_dof.py:5]: why are you adding this if it is being loaded through the proxy?

Comment (duchaoyu) [reply scripts/test_dof.py]: True...

File: ui/Rhino/AGS/dev/AGS_check_loadpath_cmd.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        return
+
+    proxy = sc.sticky['AGS']['proxy']
+    scene = sc.sticky['AGS']['scene']
+    form = scene.find_by_name('Form')[0]
+    force = scene.find_by_name('Force')[0]
+
+    proxy.package = 'compas_ags.ags.loadpath'
+
+    lp = proxy.compute_loadpath_proxy(form.diagram.data, force.diagram.data)
```
Comment (brgcode) [inline ui/Rhino/AGS/dev/AGS_check_loadpath_cmd.py:26]: with the recent updates to compas it should not be necessary anymore to use a `_proxy` wrapper for this. it should be possible to use `compute_loadpath` directly (through the proxy server).

currently the only functions that require a `_proxy` wrapper are those that don't return anything but rather update the input data structures...

Comment (duchaoyu) [reply ui/Rhino/AGS/dev/AGS_check_loadpath_cmd.py]: woo... it works... 
(don't fully get why... but it works... :-D )

**Pull request merged**